k8s/namespaces/wa/wa-task-batch-reconfiguration/ithc-01.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: wa-task-batch-reconfiguration
+spec:
+  values:
+    job:
+      schedule: "*/15 * * * *"
```

Comment:
I guess itch-00.yaml and itch-01.yaml schedule values should be different, isn't it?

Reply:
fixed timings as configuration